pdu/Src/controlStateMachine_mock.c

```diff
@@ -450,6 +372,40 @@ static const CLI_Command_Definition_t controlPumpsCommandDefinition =
     1 /* Number of parameters */
 };
 
+BaseType_t fakeHeatInv(char *writeBuffer, size_t writeBufferLength,
+                       const char *commandString)
+{
+    INV_Module_A_Temp = 45.0f;
+    INV_Module_B_Temp = 45.0f;
+    INV_Module_C_Temp = 45.0f;
+    fsmSendEvent(&mainFsmHandle, EV_EM_Enable, portMAX_DELAY);
```

Comment:
If you transition to EM Enable right after faking the temperature, thats just going to turn the MC on and overwrite that with the real temperature readings isn't it?